tests/e2e/websocket/websocket_ssl.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <algorithm>
+#include <chrono>
+#include <cstdint>
+#include <iostream>
+#include <memory>
+#include <string>
+#include <string_view>
+#include <thread>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+#include <fmt/core.h>
+#include <gflags/gflags.h>
+#include <spdlog/spdlog.h>
+#include <unistd.h>
+
+#include "common.hpp"
+#include "utils/logging.hpp"
+
+DEFINE_uint64(bolt_port, 7687, "Bolt port");
+
+class WebsocketSSLClient {
+ public:
+  WebsocketSSLClient() { session_ = {std::make_shared<Session<true>>(ioc_, ctx_, received_messages_)}; }
```

Comment:
```suggestion
  WebsocketSSLClient() { session_ = std::make_shared<Session<true>>(ioc_, ctx_, received_messages_); }
```
What is going on here with the brackets?